vhdbuilder/packer/components.json

```diff
@@ -62,6 +62,7 @@
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/coredns:*",
       "versions": [
```

Comment:
Remove the 1.5.0?

Reply:
only 1.5.0 or the versions below 1.5.0 too?

Reply:
And below. I think we use 1.6.6 for all cluster versions. We should be able to remove 1.6.5 as well.
Can you also add 1.7.1 in case there is any issue with 1.8.3, we can switch quickly.

Reply:
We need to build the VHD tonight so merging for you

Reply:
https://github.com/Azure/AgentBaker/pull/695/files

Reply:
@xuto2 Thanks Tony for the help!